salt/returners/smtp_return.py

```diff
@@ -226,14 +238,14 @@ def returner(ret):
                'Subject: {3}\r\n'
                '\r\n'
                '{4}').format(from_addr,
-                             to_addrs,
+                             ', '.join(to_addrs),
                              formatdate(localtime=True),
                              subject,
                              content)
 
     log.debug('smtp_return: Connecting to the server...')
     server = smtplib.SMTP(host, int(port))
-    server.set_debuglevel = 'debug'
+    server.set_debuglevel(1)  # enable logging SMTP session
```

Comment:
@cachedout Hmm... Good question. I haven't tested SMTP auth, but here I just fixed the *supposed* way of how "debug" mode should be enabled, because it didn't work at all before.

I've checked out `smtplib` code, and you're totally right, it logs `AUTH` commands if `debuglevel` value is more than `0`. And I think here we can safely enable session logging right after the authentication passed. I will edit the code accordingly.

Thanks for this comment, your concern is very reasonable!